usmqe/web/tendrl/mainpage/clusters/cluster_list/models.py

```diff
@@ -0,0 +1,84 @@
+"""
+Common page models for clusters.
+"""
+
+
+from webstr.core import WebstrModel, By, PageElement
+from webstr.common.form import models as form
+import webstr.patternfly.contentviews.models as contentviews
+
+
+LOCATION = '/#/cluster'
+
+
+class ClustersMenuModel(WebstrModel):
+    """
+    Clusters page top menu
+    """
+    header = PageElement(by=By.XPATH, locator="//h1[text()='Clusters']")
+    search = form.TextInput(
+        by=By.XPATH,
+        locator="//*[@placeholder='Search'][@ng-model='cluster.search']")
+    import_btn = form.Button(By.XPATH, '//button[@ng-click="cluster.importCluster()"]')
+
+
+class ClusterListModel(contentviews.ListViewModel):
+    """ list of clusters with common cluster elements """
+
+
+class ClusterRowModel(contentviews.ListViewRowModel):
+    """
+    Row in Cluster table model.
+    """
+#   TODO
```

Comment:
When this part can be uncommented?  Could you please add here any link to issue, specification, jira?